frontend/packages/kubevirt-plugin/src/selectors/vm-template/selectors.ts

```diff
@@ -132,3 +134,27 @@ export const getFlavors = (vm: VMLikeEntityKind, templates: TemplateKind[]) => {
 
   return sortedFlavors;
 };
+
+export const getRelevantTemplates = (
+  commonTemplates: TemplateKind[],
+  os: string,
+  workloadProfile: string,
+  flavor: string,
+) => {
+  if (!commonTemplates || commonTemplates.length === 0 || !os || !workloadProfile) {
```

Comment:
Nitpick: I think it's nice to allow the logic of _"if os is not defined => allow all possible os"_ 
so in my view we should do:
``` js
 if (!commonTemplates || commonTemplates.length === 0) {
```
and let the checks in L150 and L151 implement the above logic.